playground/index.html

```diff
@@ -93,19 +93,15 @@
     }
 
     input, select {
-      font-family: mono, 'Iosevka Web';
       margin: 0 0.5em 0 0.5em;
     }
 
-    textarea {
-      font-family: mono, 'Iosevka Web';
-    }
-
```

Comment:
This is actually needed: it sets font `mono` (default monospace) before Iosevka (which is a fallback option). Can you remove just Iosevka part?

Reply:
np, returned `mono`